polyschedule/image_tools/generate_algorythms.py

```diff
@@ -0,0 +1,191 @@
+import io
+from abc import ABC, abstractmethod
+
+from PIL import Image, ImageFont, ImageDraw
+
+import polyschedule.dates as dates
+
+
+class _FontStyle:
+    HEADING = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 35)
+    LABEL = ImageFont.truetype('./fonts/Comfortaa-Light.ttf', 22)
+    NORMAL = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 25)
+    SMALL = ImageFont.truetype('./fonts/Comfortaa-Regular.ttf', 18)
+
+
+class _Color:
+    BLACK = (0, 0, 0, 255)
+    GRAY = (130, 130, 130, 255)
+    GREEN = (74, 179, 76, 255)
+
+
+class _Background:
```

Comment:
As well as with the `_Color`
Moreover, the name can be better:
```suggestion
class BackgroundImages:
```